api_app/mixins.py

```diff
@@ -681,3 +682,60 @@ def authentication_header(self) -> dict:
             return {"Auth-Key": self._service_api_key}
 
         return {}
+
+
+class JoeSandboxMixin:
```

Comment:
I think that this should be better put in api_app/analyzers_manager/mixin.py and not here. 

I get that there is also the VT mixin but its position is wrong too here. Could you take this chance and move it too there? thanks

Reply:
yep, the ones related to analyzers only. Cause now they are in the wrong hierarchy level